.editorconfig

```diff
@@ -0,0 +1,16 @@
+# top level config
+root = true
+
+# All files:
+# utf-8
+# Unix-style newlines
+# Remove end-of-file newlines
+# Soft indents (2 spaces)
+# Don't trim trailing whitespace
```

Comment:
@anowlcalledjosh Can you explain the circumstances under which it would be useful to end a jade/pug line with whitespace? If we do allow that, we'll want to document why on the wiki so that we don't get questions about the inconsistency.